cffi.asd

```diff
@@ -65,11 +65,28 @@
      (:file "foreign-vars")
      (:file "features")))))
 
-(defmethod operation-done-p ((o test-op) (c (eql (find-system :cffi))))
-  nil)
-
-(defmethod perform ((o test-op) (c (eql (find-system :cffi))))
-  (operate 'asdf:load-op :cffi-tests)
-  (operate 'asdf:test-op :cffi-tests))
+(defsystem :cffi/c2ffi
```

Comment:
You mention that this change _uses the cffi/c2ffi asdf feature and includes the system in the main cffi.asd file_. What feature is that?